.github/workflows/ci.yml

```diff
@@ -0,0 +1,170 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+
+  test:
+    runs-on: ${{ matrix.os }}
+    name: test${{ matrix.name-suffix }} (${{ matrix.os }}, ${{ matrix.build-type }})
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+        build-type: [Release, Debug]
+        openmp: [false]
+        name-suffix: [""]
+        include:
+          - build-type: Debug
+    env:
+      CMAKE_FLAGS: -DOVF_BUILD_PYTHON_BINDINGS=ON -DOVF_BUILD_TEST=ON
+      BUILD_TYPE: ${{ matrix.build-type }}
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Install necessary packages
+      run: |
+        python -m pip install --upgrade pip
+        python -m pip install numpy
+
+    - name: 📁 Create build folder
+      run: cmake -E make_directory ${{runner.workspace}}/build
+
+    - name: ⚙ Configure
+      shell: bash
+      working-directory: ${{runner.workspace}}/build
+      run: cmake $GITHUB_WORKSPACE -DCMAKE_BUILD_TYPE=$BUILD_TYPE $CMAKE_FLAGS
```

Comment:
I think on the configure step, you could add
```bash
if [ "${{ matrix.os }}" == "ubuntu-latest" ]; then export CMAKE_FLAGS=$CMAKE_FLAGS -DOVF_BUILD_FORTRAN_BINDINGS=ON; fi
```
or have different configure-steps for Ubuntu and others, like here: https://github.com/GPMueller/imgui-app/blob/main/.github/workflows/ci.yml#L21-L25